### ISSUE TITLE
[TextInputField] Suffix content description

### ISSUE THREAD

**Issue by ArjunAchatz** [material-components/material-components-android#1401]
Current we are using the suffix feature that was added recently, however it doesn't read when using talkback. Can we add this ?

**Comment by matpag** [comment]
I've performed some test with Talkback.

If I add the suffix text to the TextInputLayout, when I click on the `TextInputLayout` and the field gain focus, it reads the suffix.

Like this (took from the catalog app):

```
<com.google.android.material.textfield.TextInputLayout
	style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
	android:layout_width="match_parent"
	android:layout_height="wrap_content"
	android:layout_margin="4dp"
	app:suffixText="SUFFIX TEXT"
	app:helperText="@string/cat_textfield_outlined_helper_text"
	app:helperTextEnabled="true"
	app:placeholderText="@string/cat_textfield_placeholder_text">
	
  <com.google.android.material.textfield.TextInputEditText
	  android:layout_width="match_parent"
	  android:layout_height="wrap_content"/>
	  
</com.google.android.material.textfield.TextInputLayout>
```
